build.sbt

```
@@ -4,12 +4,27 @@ val scalatestVersion = "3.2.9"
 val fs2Version = "3.8.0"
 val jqVersion = "0.1.0-SNAPSHOT"
```

```suggestion
val jqVersion = "0.1.0"
```

TODO: Change this line whenever jqscala gets its first release.